python/hsfs/feature_view.py

```diff
@@ -3734,57 +3855,197 @@ def enable_logging(self) -> None:
             # enable logging
             feature_view.enable_logging()
             ```
+
+        !!! example "Enable feature logging and add extra log columns"
+            ```python
+            # get feature store instance
+            fs = ...
+
+            # get feature view instance
+            feature_view = fs.get_feature_view(...)
+
+            # enable logging with two extra log columns
+            feature_view.enable_logging(extra_log_columns=[{"name": "logging_col_1", "type": "string"},
```

Comment:
I think restricting to list of features is better. In this case, we don't need to consider prefix and implement the matching logic.  
But if you think it is good to have more options, then it would be simpler like this  `{"logging_col_1": "string", "logging_col_2": "int"}`, but as mentioned, if type can be autofilled, then, list of feature name is better.